Voycar.Api.Web.Tests.Unit/Members/Endpoints/Put/Personal/Endpoint.cs

```diff
@@ -63,28 +67,51 @@ public async Task Put_Request_Throws_ValidationsFailure_DueToNullUser()
 
     [Fact]
     public async Task Put_Request_Throws_ValidationsFailure_DueToNullMember()
+    {
+        // Arrange
+        var ep = this.SetupEndpoint();
+        var user = new User { Email = "test@test.de", PasswordHash = "notsafe987", MemberId = null };
+
+
+        A.CallTo(() => this.FakeUserRepository.Retrieve(this.Request.UserId)).Returns(user);
+
+        // Act
+        async Task Act() => await ep.HandleAsync(this.Request, default);
+
+        // Assert
+        var exception = await Assert.ThrowsAnyAsync<ValidationFailureException>(Act);
+        Assert.NotNull(exception);
+        Assert.Equal("ThrowError() called! - User is not a member", exception.Message);
+    }
+
+
+    [Fact]
+    public async Task Put_Request_Throws_ValidationsFailure_DueToNullPlan()
     {
         // Arrange
         var ep = this.SetupEndpoint();
         var user = new User
         {
             Email = "test@test.de",
             PasswordHash = "notsafe987",
-            MemberId = null
+            MemberId = new Guid("E610101A-ED58-4A2F-8DEB-9FEAAEDD6B7C")
         };
-
+        var member = new Member { Id = (Guid)user.MemberId };
```

Comment:
```suggestion
        var member = new Member { Id = user.MemberId.Value };
```
Das sollte dasselbe machen, aber schöner